src/main/java/org/breedinginsight/services/validators/TraitValidatorError.java

```diff
@@ -67,7 +67,7 @@ public ValidationError getMissingScaleDataTypeMsg() {
 
     @Override
     public ValidationError getMissingObsVarNameMsg() {
-        return new ValidationError("observationVariableName", "Missing observation variable name", HttpStatus.BAD_REQUEST);
+        return new ValidationError("name", "Missing Name", HttpStatus.BAD_REQUEST);
```

Comment:
Should this be capitalized?

Reply:
Same question for entity and attribute in here and TraitValidationError.

Reply:
Should trait traitDescription, programObservationLevel.name, and method.formula be updated?